### PR TITLE
added more items to slot highlight

### PR DESCRIPTION
**Pull request by alon1396**

### CODE REVIEW

File: src/main/kotlin/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt [hunk 1]
```diff
@@ -88,7 +88,9 @@ class ItemFeatures {
                         if (stack.displayName.contains("Health Potion")) event.slot highlight Color(255, 225, 30, 255)
                         else if (stack.displayName.contains("Mimic Fragment") || stack.displayName.contains("Training Weights") || stack.displayName.contains(
                                 "Journal Entry"
-                            ) || stack.displayName.contains("Defuse Kit")
+                            ) || stack.displayName.contains("Defuse Kit") || stack.displayName.contains("Lever") ||
```
Comment (Sychic) [inline src/main/kotlin/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt]: use a contains any

**Review by Sychic — CHANGES_REQUESTED** [review]

**Review by Sychic — APPROVED** [review]
Could also combine the current checks into the contains any too

**Review by Sychic — APPROVED** [review]

File: src/main/kotlin/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt [hunk 2]
```diff
@@ -86,9 +86,9 @@ class ItemFeatures {
                     if (event.slot.hasStack) {
                         val stack = event.slot.stack
                         if (stack.displayName.contains("Health Potion")) event.slot highlight Color(255, 225, 30, 255)
-                        else if (stack.displayName.contains("Mimic Fragment") || stack.displayName.contains("Training Weights") || stack.displayName.contains(
-                                "Journal Entry"
-                            ) || StringUtils.containsAny(stack.displayName, "Defuse Kit", "Lever", "Torch", "Stone", "Button", "Tripwire Hook")
+                        else if (StringUtils.containsAny(stack.displayName, "Defuse Kit", "Lever", "Torch", 
+                                                         "Stone", "Button", "Tripwire Hook", "Journal Entry",
```
Comment (AzuredBlue) [inline src/main/kotlin/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt]: why does it highlight stone too it's supposed to be stone buttons

Comment (alon1396) [reply src/main/kotlin/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt]: good point

**Review by My-Name-Is-Jeff — APPROVED** [review]

**Pull request merged**